modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_logical-volume-manager-storage"]
+= Integrating software RAID arrays with {lvms}
+
+You can create the Redundant Array of Independent Disks (RAID) array by using the `mdadm` utility, and integrate the RAID array with {lvms}. Logical Volume Manager (LVM) does not support creating a software RAID.
+
+You can integrate the RAID array with {lvms} while creating the `LVMCluster` custom resource (CR).
+
+.Prerequisites
+
+* You created a software RAID during the {product-title} installation.
++
+[IMPORTANT]
+====
+You can only create the RAID array during the installation of {product-title}.
+====
+
+* You have installed {lvms}.
+
+.Procedure
+
+. Open the `LVMCluster` CR YAML file.
++
+[NOTE]
+====
+If you have created the `LVMCluster` CR, edit the existing `LVMCluster` CR YAML file.
+====
+
+. Add the path to the RAID array in the `deviceSelector` field of the `LVMCluster` CR YAML file.
++
+.Example of RAID array paths in deviceSelector field
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: my-lvmcluster
+spec:
+  storage:
+    deviceClasses:
+# ...
+      deviceSelector:  <1>
+        paths:
+        - /dev/md/md-var  <2>
+        - /dev/md0  <3>
+        optionalPaths:
+# ...
+----
+<1> Contains the paths to the devices that are used to create the LVM volume group. You can specify the device paths in the `paths` field, the `optionalPaths` field, or both. You cannot remove or replace a device after adding it to the LVM volume group.
+<2> Example of a path to the RAID array that was created by using the `mdadm` utility. In this example, `md-var` is the RAID array.
+<3> In this device path example, `md0` is the RAID array
+
+. Save the `LVMCluster` CR YAML file.
```

Comment:
Does saving the file apply the changes to the cluster? Can you specify what happens at this point?

Reply:
@jakobmoellerdev 
Could you please confirm if saving the file is enough or running a command to apply the file is required?